doc/sphinx/source/conf.py

```diff
@@ -73,11 +73,11 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.21'
+version = '0.22'
 
 # The Documentation full version, including alpha/beta/rc tags. Some features
 # available in the latest code will not necessarily be documented first
-release = '0.21.0'
+release = '0.22-doc'
```

Comment:
What does `-doc` mean here?

Reply:
I think this because we're not actually going to get this doc published on the NervanaSystems site, we're moving straight to 0.23 after this.